Doc/library/functions.rst

```diff
@@ -692,6 +692,12 @@ are always available.  They are listed here in alphabetical order.
    ``x.foobar``.  If the named attribute does not exist, *default* is returned if
    provided, otherwise :exc:`AttributeError` is raised.
 
+   .. note::
+
+      Since :ref:`private name mangling <private-name-mangling>` happens at
+      compilation time, attributes with two leading underscores are not found
+      by :func:`getattr` unless the mangled name is used.
+
```

Comment:
Please change to something like
> Since [...] only happens at compilation time, one must manually mangle a private attribute's name in order to retrieve with :func:`getattr`.